core/src/test/java/org/apache/calcite/test/JdbcTest.java

```diff
@@ -8217,6 +8217,72 @@ private void checkGetTimestamp(Connection con) throws SQLException {
         .returns("EXPR$0=[1, 1.1]\n");
   }
 
+  @Test void testMultiLevelDecorrelation() throws Exception {
```

Comment:
Should this function have a JavaDoc describing the issue fixed?

Reply:
Yeah sure, I can add with the reference to the CALCITE JIRA.